src/kaleidoscope/plugin/Qukeys/KeyAddrEventQueue.h

```diff
@@ -45,7 +45,7 @@ class KeyAddrEventQueue {
  private:
   uint8_t    length_{0};
   KeyAddr    addrs_[_capacity];
-  _Timestamp timestamps_[_capacity];
+  _Timestamp timestamps_[_capacity]; // NOLINT(runtime/arrays)
```

Comment:
`_capacity` is a template argument forced into a local constexpr - it is known at compile-time, and so is `_Timestamp`. This is a false-positive, so tell cpplint to ignore it.

Reply:
Out of curiosity, what was the warning?